spec/features/users/index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page' do
+  before :each do
+    load_test_data
+  end
+
+  #  When a user visits the root path they should be on the landing page ('/') which includes:
+  #  Title of Application
+  #  Button to Create a New User
+  #  List of Existing Users which links to the users dashboard
+  #  Link to go back to the landing page (this link will be present at the top of all pages)
+
+  it 'displays a title' do
+    visit(root_path)
+    expect(page).to have_content('Viewing Party Lite')
+  end
+
+  it 'displays a button to create a new user' do
+    visit(root_path)
+    expect(page).to have_link('Create New User')
```

Comment:
Include the link you expect the link to go to, ie: `have_link('Create New User', href: path)` 
This will test the link has a working route associated with it

This is another example of when the `register_user_path` alias could be used (see comment on user index view)